snap/snapcraft.yaml

```diff
@@ -19,39 +19,101 @@ slots:
 
 apps:
   fractal:
-    extensions: [gnome-3-38]
+    extensions: [gnome]
     command: usr/bin/fractal
     common-id: org.gnome.Fractal
     desktop: usr/share/applications/org.gnome.Fractal.desktop
     plugs:
-      - audio-playback
       - home
-      - gsettings
-      - network
-      - opengl
-      - password-manager-service
+      - removable-media
 
 parts:
+  ffmpeg:
```

Comment:
Does fractal really use ffmpeg now?  If so, does it still use gstreamer?  Perhaps gstreamer could be removed now...

Reply:
I got it from the flatpak manifest.

https://gitlab.gnome.org/GNOME/fractal/-/blob/4.4.2/flatpak/org.gnome.Fractal-release.json#L27